spikeextractors/extractors/neoextractors/neobaseextractor.py

```diff
@@ -153,6 +153,7 @@ def get_channel_ids(self):
         return self._channel_ids
 
 
+
```

Comment:
```suggestion
```
This line was probably added by accident. To avoid this, always run `git status` to see what files have been changed before calling `git add` to include only the ones you 100% intend. In particular, adding this line is a PEP8 violation (two line break for class definitions).

Reply:
Indeed, must have been an accident. Thanks for catching it.